test/unit/spec/OktaSignIn_spec.js

```diff
@@ -355,6 +355,30 @@ Expect.describe('OktaSignIn v2 bootstrap', function () {
     return signIn.renderEl({ el: $sandbox });
   }
 
+  function setupProxyIdxResponse (options) {
```

Comment:
yeah that will work. It's just i would like to avoid unnecessary setup for this use case. And also, since we won't actually make introspect call, `setupIntrospect` doesn't seems to be a appropriate name.

Reply:
hmm, good point.